cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -274,6 +274,37 @@ public void testGenerateStatefulSetWithEmptyStorageSelector() {
         assertEquals(null, ss.getSpec().getVolumeClaimTemplates().get(0).getSpec().getSelector());
     }
 
+    @Test
+    public void testGenerateStatefulSetWithSetSizeLimit() {
+        String sizeLimit = "1Gi";
+        Kafka kafkaAssembly = new KafkaBuilder(ResourceUtils.createKafkaCluster(namespace, cluster, replicas,
+                image, healthDelay, healthTimeout, metricsCm, configuration, emptyMap()))
+                .editSpec()
+                .editKafka()
+                .withNewEphemeralStorage().withNewSizeLimit(sizeLimit).endEphemeralStorage()
+                .endKafka()
+                .endSpec()
+                .build();
```

Comment:
maybe you can use indentation for more readability? The same for the other tests.

Reply:
Updated as suggested.